src/plopp/core/node.py

```diff
@@ -24,18 +49,35 @@ class Node:
     """
 
     def __init__(self, func: Callable, *parents, **kwparents):
-        if not callable(func):
-            raise ValueError("A node can only be created using a callable func.")
-        self.func = func
-        self._id = str(uuid.uuid1())
+        func_is_callable = callable(func)
+        if func_is_callable:
+            self.func = func
+        else:
+            self.func = lambda: func
```

Comment:
Writing the conversion to dask graphs, I wish I had a way to get the input value out of an input node. Can you store `func` in an attr like `_input_data` if it is not callable? Then properties like `input_data` (getter only) and `is_input_node` would help to inspect the graph.

Reply:
Done.